### ISSUE TITLE
Proxy connection issues after updating

### ISSUE THREAD

**Issue by RafaelLinux** [telegramdesktop/tdesktop#16284]
### Steps to reproduce
1. Update to Telegram Desktop 2.7.5 beta (using corporate proxy)

### Expected behaviour
Run TD smoothly as always

### Actual behaviour
TD has a lot of connection issues:

- Show "connecting" in group/channel header intermittently.
- At same time, is shows a "shield icon" bottom left,  from time to time shows "Reconnect in X s .... Try now" even 50 seconds (some times "Try now" solves problem for some seconds, but if I move to a new location (upper or lower) or change to another channel group, same message "Reconnect ..." appears again and again-  

### Configuration
openSUSE 15.2 using corporative proxy 
Telegram Desktop 2.7.5 beta, configured to use proxy (as always I do from years ago)
https://desktop.telegram.org/ tar.xz source
No theme

**Comment by ilya-fedin** [comment]
:thinking: I tried to activate libproxy, it should get proxy settings from your DE now

**Comment by ilya-fedin** [comment]
You can run Telegram with `_PX_DEBUG=1` to see which settings it uses

**Comment by RafaelLinux** [comment]
This is the ouput
```
./Telegram _PX_DEBUG=1
[ALSOFT] (EE) Failed to set real-time priority for thread: Operación no permitida (1)
error: : cannot open
error: : cannot open
error: : cannot open
  OpenType support missing for "DAOpenSansSemibold", script 11
  OpenType support missing for "DAOpenSansSemibold", script 14

```
But it seems doesn't log any of the connection issues related. Each time Telegram is showing "Reconnect in X  seconds" or I click "Try now" button, no output at all in console (I thought Telegram will show something)  :(

**Comment by ilya-fedin** [comment]
`_PX_DEBUG=1 ./Telegram` :roll_eyes:

**Comment by ilya-fedin** [comment]
> I thought Telegram will show something

Telegram doesn't log anything own to terminal, it has a `log.txt` in the data folder.

**Comment by RafaelLinux** [comment]
And where is that data folder? 
"$user/.TelegramDesktop/" folder doesn't have any log file despite I launch Telegram using _PX_DEBUG=1 ./Telegram

**Comment by ilya-fedin** [comment]
libproxy writes messages to terminal. I don't think Telegram logs will contain anything interesting.

**Comment by ilya-fedin** [comment]
> what can I inspect to guess what's happening?

You should run `_PX_DEBUG=1 ./Telegram` and see libproxy's debug messages like `Config is:`, you should paste them here.

> Why did you mention that data folder????

Because you asked about Telegram log

**Comment by RafaelLinux** [comment]
> > what can I inspect to guess what's happening?
> 
> You should run `_PX_DEBUG=1 ./Telegram` and see libproxy's debug messages like `Config is:`, you should paste them here.

You talk about "libproxy's debug messages" but didn't mention what log file to look in. I watched journald, system and even xorg log, but no lines mentioning "libproxy". Could you be more specific about which and where is the log I should look for to find some message referencing "libproxy".

> > Why did you mention that data folder????
> 
> Because you asked about Telegram log
Well, I asked about a "data folder" cause you wrote "Telegram doesn't log anything own to terminal, it has a log.txt **in the data folder**" that I can't find to look for a "log.txt"  Telegram file  ;)

Thank you

**Comment by ilya-fedin** [comment]
> You talk about "libproxy's debug messages" but didn't mention what log file to look in.

I [mentioned](https://github.com/telegramdesktop/tdesktop/issues/16284#issuecomment-834257787):

> libproxy writes messages to terminal

**Comment by RafaelLinux** [comment]
This is going nowhere. Maybe you are guessing I know how to get useful info to show you from a developer or console user, and that's not the way we could help each other.
I think this is your point of view cause clearly, in you first instructions "https://github.com/telegramdesktop/tdesktop/issues/16284#issuecomment-832701877" you only told me _"You can run Telegram with _PX_DEBUG=1 to see which settings it uses"_ and I guessed  __PX_DEBUG=1_ was a Telegram parameter that I should enter at console. But that was not the case. It was not a parameter for Telegram.

Conclusion: If you are not only "a little" more specific, this seems we are talking a lot of blather.

Thank you anyway

**Comment by ilya-fedin** [comment]
_PX_DEBUG is an environment variable, you should run Telegram like `_PX_DEBUG=1 ./Telegram` (if you're using bash) and it should give a lot of lines like `Config is:` to terminal output. I don't what could be easier than that. :roll_eyes:

**Comment by RafaelLinux** [comment]
But I put the output of that command 11 posts ago!!!! And it didn't help!!! In fact, after one hour, no changes in that output!!!! 

```
~/bin/Telegram> _PX_DEBUG=1 ./Telegram
[ALSOFT] (EE) Failed to set real-time priority for thread: Operación no permitida (1)
error: : cannot open
error: : cannot open
error: : cannot open
  OpenType support missing for "DAOpenSansSemibold", script 11
  OpenType support missing for "DAOpenSansSemibold", script 14
QNetworkReplyImplPrivate::error: Internal problem, this method must only be called once.
  OpenType support missing for "DAOpenSansRegular", script 11
  OpenType support missing for "DAOpenSansSemibold", script 15
QNetworkReplyImplPrivate::error: Internal problem, this method must only be called once.
QNetworkReplyImplPrivate::error: Internal problem, this method must only be called once.
```

**Comment by ilya-fedin** [comment]
Well, that's really weird. Does `env _PX_DEBUG=1 ./Telegram` add any lines or not?

**Comment by ilya-fedin** [comment]
> But I put the output of that command 11 posts ago!!!!

I see only output of `./Telegram _PX_DEBUG=1`, but not of `_PX_DEBUG=1 ./Telegram` 11 posts ago...

**Comment by ilya-fedin** [comment]
PS: you shouldn't wait one hour, the lines should appear soon after launch

**Comment by ilya-fedin** [comment]
> openSUSE 15.2 using corporative proxy
> Telegram Desktop 2.7.5 beta, configured to use proxy (as always I do from years ago)

Does this mean you have configured the proxy via system settings or via Telegram settings? I assumed the first.

**Comment by RafaelLinux** [comment]
It's clear you was only quickly answering instead really taking time to reading my posts.
I posted bugs and issues to a lot of another software bugs tracking services since years ago, and I never found someone that made me feel that my efforts to share an issue were a waste of time, cause who answer my is like not taking attention to what he write nor about I write him.
You really didn't take time even to read my posts contents, but only to quick replying as if you don't care about.

Anyway, I'll answer your last question: I tried both proxies configuration, system settings (was default before this issue appeared after last update) and Telegram settings, with exactly same result.

I give up and will make it easier for both, and I'll close this issue. 

Really I feeled ingored with your answers.

Thank you.
